### PR TITLE
[added] test running for specific packages

### PR DESCRIPTION
**Pull request by tofuness**
Relevant issue #112 

* `yarn run test -- --only runtime` (from root dir) - only runs the tests for the `runtime` package
* `node scripts/test --only runtime` - same things as above
* `yarn run test -- --grep time` (from root dir) - runs the tests for the packages that have `time` in their name.
* `node scripts/test --grep time` - same thing as above

I placed the `test.js` file in a separate directory called `scripts` (since more scripts will probably be added in the future).

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -1,45 +1,74 @@
...
-    port: 9876,
-    colors: true,
-    logLevel: config.LOG_INFO,
-    autoWatch: true,
-    singleRun: false,
-    concurrency: Infinity,
-  });
+      webpackMiddleware: {
+        stats: {
+          chunks: false,
```
Comment (tofuness) [inline karma.conf.js:41]: Suppresses some console.log messages when bundle is built

File: karma.conf.js [hunk 2]
```diff
@@ -1,45 +1,74 @@
...
-  config.set({
-    basePath: '',
-    frameworks: ['mocha'],
-    browsers: ['Chrome', 'ChromeCanary', 'Safari', 'Firefox'],
-    reporters: ['progress'],
-    client: {
-      mocha: {
-        reporter: 'html',
+  // Default config for all packages
+  let opts = {
```
Comment (appsforartists) [inline karma.conf.js]: In general, let's prefer descriptive names over terse; perhaps `defaultConfig` here?

File: karma.conf.js (hunk 1 above)
Comment (appsforartists) [reply karma.conf.js:41]: Sounds like a great comment to leave in the file itself. 😃

File: karma.conf.js [hunk 3]
```diff
@@ -1,45 +1,74 @@
...
+      logLevel: config.LOG_INFO,
+      autoWatch: true,
+      singleRun: false,
+      concurrency: Infinity,
+  };
+
+  const argv = minimist(process.argv);
+  if (argv.only) {
+    // Run tests for a specific package
+    if (!fs.existsSync(`./packages/${argv.only}`)) {
```
Comment (appsforartists) [inline karma.conf.js]: Style nit: template literals are easier to read if you pad the inside with spaces: `${ argv.only }`

**Comment by appsforartists** [comment]
Yay!  Thanks for the quick turnaround.

How many scripts do you think we'll have?  I can foresee `build` and `test`.  `devServer` is possible, but unlikely in the root.

If it's just 2 or 3, do you think it's worth putting them in their own folder?  Right now, it's really obvious to newcomers that all source must be in the `packages` folder; that will be less clear if we have other similarly-named folders.  If we keep it, maybe we call it something that's less likely to be confused with source code, like `tools`.  What do you think?

I was tempted to suggest unifying `only` and `grep`, until I imagined a situation where you wanted to test a package called `streams` but couldn't exclude another called `style-streams`.

Thanks again for owning this!

**Review by appsforartists — APPROVED** [review]

**Comment by appsforartists** [comment]
Moved to http://codereview.cc/D2320

**Pull request closed without merging**